app/src/screens/SignIn/__tests__/SignIn.test.tsx

```diff
@@ -27,4 +28,22 @@ describe('<SignIn />', () => {
       ),
     ).toBeOnTheScreen();
   });
+
+  it('should show the accessToken error message', () => {
+    const errorMessage = 'wrong email';
+    const initialState = {
+      voice: {
+        accessToken: {
+          status: 'rejected',
+          error: { message: errorMessage },
+        },
+      },
+    };
+    const mockStore = configureStore();
+    wrapper = ({ children }) => (
```

Comment:
Do we want to put this in a `beforeEach` hook like our other component tests?

Reply:
This file does have the `beforeEach` `store` hook. However, to mock the state, I need to use the `mock-store`.  We can't put it in the `beforeEach` because the `mock state` it will be different for each test case.

Reply:
Gotcha. There's actually some substantial work with component testing and using mock stores vs our actual store that I think we need to do. I don't think we can get away with mocking our stores using the mock store library anymore. While working on the audio device component tests, I can't dispatch thunks because the mock store expects plain objects only. Out of scope for this ticket and PR, but something to keep in mind.

Reply:
For now, until we can revisit all of the component tests, I can live with the current implementation in this file.

Reply:
got it. I replaced the `mock-store` with the actual store.